### DISCUSSION TITLE
why

### DISCUSSION THREAD

**Discussion by GabubuAvailable** [microsoft/sudo#105] [General]
just why? :)

**Comment by hwalker56** [comment]
but really like why

**Comment by atif-u** [comment]
wawa

**Comment by AvogatoWizardWhisker** [comment]
I can answer that. Sudo for Windows can be beneficial to IT pros that are accustomed to using `sudo` in Linux to run commands with elevated privileges. Bringing `sudo` to Windows provides familiar and ergonomic solution for users who switch between OSes. Sudo for Windows is also convenient when running commands directly from an elevated console session, so you don't have to open another console window and run it as admin. Sudo for Windows is also flexible as it offers different configurations to suit your needs: `--new-Window` `--disable-input` and `--inline` which [they explained their purpose in the following documentation on how to configure it](https://learn.microsoft.com/en-us/windows/sudo/#how-to-configure-sudo-for-windows). In terms of security, Sudo for Windows provides convenience and with security considerations. Users need to be aware of [potential risks and configure `sudo` appropriately.](https://learn.microsoft.com/en-us/windows/sudo/#security-considerations).

TL;DR: Sudo for Windows is beneficial for users familiar Linux, offering a convenient and ergonomic way to run elevated commands without opening a new console window. It provides flexibility with different configurations, and it requires users to be aware of security considerations

**Reply by GabubuAvailable** [reply]
That makes sense now. Thank you for your reply! 🙂
